oneview_redfish_toolkit/services/logging_service.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import logging
+import threading
+
+COUNTER_LOGGER_NAME = 'qtty'
+
+
+def debug(logger_name, *texts):
+    if logging.getLogger().isEnabledFor(logging.DEBUG):
+        msg = "Thread {}: ".format(threading.get_ident())
+        if len(texts) > 1:
+            msg += "\n"
+
+        msg += "\n".join(texts)
```

Comment:
Why not log in the same line? It seems easier to convert later in case we want to.

Reply:
done